src/plugins/stacked-barchart/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "fun-stacked-barchart",
```

Comment:
I would prefer prefixing the plugin name with `potsie` instead of FUN as it's more project than organization-specific.

Reply:
Good point) Thanks) Renamed with `potsie`)